.circleci/config.yml

```diff
@@ -22,9 +22,11 @@ jobs:
       - image: circleci/python:3.6
     steps:
       - checkout
-      - run: pip3 install --upgrade mypy black
-      - run: black --check .
-      - run: mypy -p "brainframe.cli"
+      - run: "curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py
+          | python3 - --version 1.1.0b2"
+      - run: poetry install --no-root
+      - run: poetry run black --check .
+      - run: poetry run mypy -p "brainframe.cli"
```

Comment:
should we use `--strict`?

Reply:
I tried `--strict` out and, in my opinion, it was too onerous. It requires you to annotate types everywhere, which I started doing, but it ended up with me annotating a lot of trivial situations that didn't seem like they were worth the verbosity.